force-app/main/default/classes/SOQL.cls

```diff
@@ -39,9 +42,13 @@ public with sharing class SOQL {
      * @return A list of Contact records.
      * Update the corresponding test class and testGetContactsByFirstName method to cover this method
      */
-    //[public | private ] [static] data_type method_name (input parameters) {
+    public static List<Contact> getContactsByFirstName () {
     //  The body of the method
-    //}
+    List<Contact> contacts = [SELECT Id, FirstName, Name 
+                               FROM Contact 
+                               WHERE FirstName = 'John'];
+    return contacts;
```

Comment:
How could we write this method to take an argument of String firstName and return ANY Contacts with that firstName value we pass in?